.github/workflows/release.yml

```diff
@@ -49,9 +51,45 @@ jobs:
         run: |
           echo "TAG=${GITHUB_REF##*/}" >> $GITHUB_ENV
 
+      - name: Count features
+        if: ${{ env.TAG == 'latest'}}
+        # get all commits in develop that aren't in master, find any that start with feat: or feat(*):, then count them
+        # 1. `git tag --sort=-version:refname --no-contains HEAD`
+        #    - get a list of all tags that do NOT contain out current HEAD commit
+        #    - sort by version name
```

Comment:
I didn't know about `--sort=-version:refname`.  N.B, the Sort order defaults to the value configured for the `tag.sort` variable from the applied `.gitconfig` or lexicographic order otherwise.  Mentioning this in case anyone on the team has that variable defined, they may see different results. 

>       --sort=<key>
           Sort based on the key given. Prefix - to sort in descending order of the value. You may use the --sort=<key> option multiple times, in which case the last key becomes the
           primary key. Also supports "version:refname" or "v:refname" (tag names are treated as versions). The "version:refname" sort order can also be affected by the
           "versionsort.suffix" configuration variable. The keys supported are the same as those in git for-each-ref. Sort order defaults to the value configured for the tag.sort
           variable if it exists, or lexicographic order otherwise. See git-config(1).